android/src/com/mapswithme/maps/routing/RoutingBottomMenuController.java

```diff
@@ -266,15 +266,10 @@ private void showRouteAltitudeChart()
       UiUtils.show(mAltitudeChart);
       String meter = mAltitudeDifference.getResources().getString(R.string.meter);
       String foot = mAltitudeDifference.getResources().getString(R.string.foot);
```

Comment:
This call is unnecessary in the metric system, and the previous call is useless in the imperial one. Please rewrite the code without unnecessary function calls. We want to have _the fastest_ app that eats _less_ battery 😉

Reply:
Removed unnecessary function calls d80c3a0